flipperdevices/flipperzero-firmware
# Loader: Warn about missing SD card for main apps

# What's new

- If SD card missing or unmounted, show appropriate message when launching main menu apps instead of "update firmware"

# Verification 

- Remove SD card
- Launch main menu app
- Observe relevant error message

# Checklist (For Reviewer)
